src/components/atoms/SfBadge/SfBadge.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../../css/variables';
+
+
+$sf-badge--text-color: $c-white;
```

Comment:
one more thing - we don't use sf- prefix in scss vars

Reply:
Ok. I will update my PR to delete fixed width and prefix vars